sphinx_rtd_theme/static/js/versions.js_t

```diff
@@ -8,10 +8,14 @@ if (themeFlyoutDisplay === "attached") {
       return "";
     }
 
+    // Prepend the current language to the options on the selector
+    let languages = config.projects.translations.concat(config.projects.current);
+    languages = languages.sort((a, b) => a.language.name.localeCompare(b.language.name));
```

Comment:
I have a feeling `sort` is not the correct operation here. I'm guessing this might randomly sort the list of locales after the first (current) locale. Using sort to only target one element in the list will resort/unsort/? the rest of the elements.

Is there a reason you can't use `insert(0, ..)` here?

Reply:
Meh, the comment is wrong, but the code is correct.

We want to sort the languages alphabetically here, no matter where we insert the current project.